modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -506,6 +506,15 @@ NOTE: The new `trigger` option can handle multiple character strings as the trig
 
 For more information, visit the updated xref:autocompleter.adoc[Autocompleter] documentation.
 
+=== The open-source `Template` plugin has been removed.
+// #TINY-10654
+
+The open-source `Template` plugin and associated config options have been removed in {productname} 7.0.
+
+Customers using the `template` plugin will need to migrate to the premium `advtemplate` which was renamed to `Templates` in {productname} 7.0, which contains all the functionality except some esoteric options.
```

Comment:
Not sure if we need to mention the product name "Templates" as I am worried it could cause more confusion then anything. Could also link the `advtemplate` docs page?
```suggestion
Customers using the `template` plugin will need to migrate to the premium `advtemplate` plugin in {productname} 7.0, which contains all the functionality except some esoteric options.
```